fixed template in places that only affect Studio

sarahkf opened the pull request:
Edited the HTML template for the Files & Uploads page in Studio to make it more accessible, following the guidelines of the linked JIRA ticket.

https://openedx.atlassian.net/browse/EDUCATOR-605
https://studio-sarahkf.sandbox.edx.org/

sarahkf commented:
jenkins run js

cahrens commented on cms/templates/asset_index.html:
```diff
@@ -35,32 +35,31 @@
+        </div>
 
-        <aside class="content-supplementary" role="complementary">
+        <aside class="content-supplementary" role="complementary" aria-label="Help adding Files and Uploads">
```
You must mark strings that a user will encounter for translation.

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html

cahrens commented on cms/templates/asset_index.html:
```diff
@@ -35,32 +35,31 @@
             <small class="subtitle">${_("Content")}</small>
-            <span class="sr">&gt; </span>${_("Files & Uploads")}
-        </h1>
+            <span class="sr">- </span>${_("Files & Uploads")}
```
Do we have to have this dash?

sarahkf replied:
@cptvitamin explained that it's purpose it to tell a person using a screen reader that `Files & Uploads` is a subset of `Content`.

cptvitamin replied:
@cahrens Originally, it was a greater than symbol, indicating the relationship between the word Content and Files & Uploads, but while the greater than symbol visually indicates that relationship to us, its not so effective when read out by a screen reader (GREATER THAN) so I suggested @sarahkf replace it with a dash, which is a better convention for indicating this relationship.

cahrens commented:
@sarahkf Please include the ticket number in the commit comment to it links with the JIRA ticket. I think I commented on this in another of your PRs, but if you aren't clear what to do, let me know.

cahrens commented:
@sarahkf I've taken a look. I have a couple comments, and could you also update the JIRA ticket to make it clear what is covered in this PR vs. EDUCATOR-661?

sarahkf commented:
@cahrens 

About the JIRA

sarahkf commented:
@cahrens I made the change in the HTML that you suggested. About linking to the JIRA ticket - I didn't see that comment until after I had made the commits on this branch, but I added it to my most recent commit so that should be good now. I also cleaned up both the new JIRA ticket and the old JIRA ticket for fixing the html templates.

cahrens commented on cms/templates/asset_index.html:
```diff
@@ -35,32 +35,31 @@
+        </div>
 
-        <aside class="content-supplementary" role="complementary">
+        <aside class="content-supplementary" role="complementary" aria-label="${_("Help adding files and uploads")}">
```
Nit: I would capitalize Files and Uploads.

cptvitamin commented on cms/templates/asset_index.html:
```diff
@@ -35,32 +35,31 @@
 
-        <nav class="nav-actions" aria-label="${_('Page Actions')}">
-            <h3 class="sr">${_("Page Actions")}</h3>
+        <div class="nav-actions" aria-label="${_('Page Actions')}">
```
@sarahkf I wasn't specific enough with my guidance here. We also want to remove the aria-label on this div. The presence of an aria-label will trigger an heuristic behavior that results in the screen reader treating this like an accessible object, which its not, and why we are converting it to a div.

sarahkf replied:
@cptvitamin This is fixed now!
